server/plugin/command.go

```diff
@@ -60,16 +64,20 @@ func validateFeatures(features []string) (bool, []string) {
 	return valid, invalidFeatures
 }
 
-func getCommand(config *Configuration) *model.Command {
-	return &model.Command{
-		Trigger:          "github",
-		DisplayName:      "GitHub",
-		Description:      "Integration with GitHub.",
```

Comment:
`DisplayName` and `Description` are not needed for plugins as they are not shown.